app/src/main/contrib/components/Responsibilities/Overview/List/ResponsibilityScenario.tsx

```diff
@@ -1,25 +1,30 @@
 import * as React from "react";
+import { connect } from 'react-redux';
+
 import {
+    Disease,
     ModellingGroup, Responsibility, ResponsibilitySetStatus,
     Touchstone
 } from "../../../../../shared/models/Generated";
-import { mainStore } from "../../../../stores/MainStore";
 import { ButtonLink } from "../../../../../shared/components/ButtonLink";
 import {CurrentEstimateSetSummary} from "./CurrentEstimateSetSummary";
+import {ContribAppState} from "../../../../reducers/contribAppReducers";
 
-interface Props {
+export interface ResponsibilityScenarioProps {
     responsibility: Responsibility;
     modellingGroup: ModellingGroup;
     touchstone: Touchstone;
     responsibilitySetStatus: ResponsibilitySetStatus;
+    diseases?: Disease[];
```

Comment:
I would prefer if we used the `PublicProps` approach we've used elsewhere. So 
```
interface ResponsibilityScenarioPublicProps {
     responsibility: Responsibility;
     modellingGroup: ModellingGroup;
     touchstone: Touchstone;
     responsibilitySetStatus: ResponsibilitySetStatus;	     
     responsibilitySetStatus: ResponsibilitySetStatus;
}

interface ResponsibilityScenarioProps  extends ResponsibilityScenarioPublicProps {
     diseases: Disease[];
}
```

Then `mapStateToProps` takes an argument of type `ResponsibilityScenarioPublicProps` and returns  `ResponsibilityScenarioProps`

Reply:
Similarly in the `TemplateLinks` component below